pkg/issuer/acme/http/http.go

```diff
@@ -139,7 +139,7 @@ func (s *Solver) Present(ctx context.Context, issuer v1.GenericIssuer, ch *cmacm
 			_, istioErr = s.ensureIstio(ctx, ch, svc.Name)
 		} else {
 			// TODO: add support for dynamic Istio support detection (start / stop the VirtualService lister as necessary)
-			istioErr = errors.New("Istio support was not detected on startup, try restarting cert-manager")
+			istioErr = errors.New("istio support was not detected on startup, try restarting cert-manager")
```

Comment:
This seems like a step backwards. Istio should be capitalized....but this Istio support has been reverted anyway, so it's moot.